server/app.js

```diff
@@ -24,26 +26,46 @@ const app = express();
 const server = http.createServer(app);
 
 const io = socketio(server, {
-	cors: {
-		origin: "*",
-	},
+  cors: {
+    origin: "*",
+  },
 });
 
+// Web Client starts a websocket connection. Each one is saved in connectedUsers list
+let connectedUsers = {};
 io.on("connection", (socket) => {
-	console.log("connected");
+  console.log("Socket connected:", socket.id);
+  connectedUsers[socket.id] = null;
+  console.log(connectedUsers);
```

Comment:
Now that you've tested, you can remove the console logs in here

Reply:
Console.logs deleted.